### DISCUSSION TITLE
sudo brew install (skip root password for fresh install automation)

### DISCUSSION THREAD

**Discussion by namluu25** [Homebrew/discussions#3545] [Casks]
I have created a script for auto fresh installing a Macs.  But some of formulae/casks requires root password to install. I have tried to run the script with ```sudo``` but Homebrew won't accept that run. Is there anyway to skip/autofill the root password?

**Comment by gromgit** [comment]
If all the password prompts you see are from `sudo`, then `SUDO_ASKPASS` is probably the direction you should take: https://stackoverflow.com/questions/12608293/how-to-setup-a-sudo-askpass-environment-variable
